httpclient/HTTPClient.cpp

```diff
@@ -19,21 +19,11 @@ long		HTTPClient::m_iConnectionTimeout = 10;
 long		HTTPClient::m_iTimeout = 90; //max, time that a download has to be finished?
 std::string	HTTPClient::m_sUserAgent = "domoticz/1.0";
 
-size_t write_curl_headerdata(void *contents, size_t size, size_t nmemb, void *userp)
+size_t write_curl_headerdata(void *contents, size_t size, size_t nmemb, void *userp) // called once for each header
 {
-	// called once for each header
 	size_t realsize = size * nmemb;
 	std::vector<std::string>* pvHeaderData = (std::vector<std::string>*)userp;
```

Comment:
If a header line is terminated with crlf (\r\n) (which it should be according RFC 1945) then this might leave \r at the end in each vector item?
Is this for 1 header line or for all headers? as i think this will only work for the first header line

Reply:
Hmm I admit only checked if new lines would occur printing them, which it doesn't stopping at \n, but you're right, stopping at \r also prevents printing them, so I'll assume it's always \r\n then (and change it to stop at \r)

This function is called on each header, not once for all headers, which is why this works.